location-service/src/main/java/com/movie/recommender/location/controller/LocationController.java

```diff
@@ -53,4 +59,17 @@ public ResponseEntity<List<CountryWithCitiesDTO>> getAllCountriesAndCities() {
         List<CountryWithCitiesDTO> result = locationService.getAllCountriesAndCities();
         return ResponseEntity.ok(result);
     }
+
+    private void triggerCrawlProcess(String action) {
```

Comment:
create a dedicated service class `RabbitMQMessageService` for handling message sending operations
move this method to the new class